<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerRealTimeCurveWidget</class>
 <widget class="QWidget" name="PowerRealTimeCurveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,7">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1" colspan="2">
      <widget class="QCheckBox" name="all_clean">
       <property name="text">
        <string>全清</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>电源屏</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="all_check">
       <property name="text">
        <string>全选</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>添加到显示列表</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>设备类型</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="displaytime">
       <property name="text">
        <string>显示时间</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="display_Overload">
       <property name="text">
        <string>显示超限路数</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="queryButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QTableView" name="tableView_2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
